### DISCUSSION TITLE
Easy Tools for Avatar and Screencast key tutor video.

### DISCUSSION THREAD

**Discussion by mac4kent** [nutti/Screencast-Keys#76] [General]
I do support you on screencast key
I do support Easy Tool for Avatar to show your screencast key too
this is my own group on Discord https://discord.gg/7PygcgmSvv
About Blender / SecondLife / OSGrid.org

**Comment by nutti** [comment]
Thanks for using Screencast Key add-on continuously.
I will support Screencast Key in the future as possible.

I have lots of other high priority tasks recently, so I could not spare lots of my time to support of this add-on.
If you want to receive more support, please consider to contribute the patch or become my sponsor.
https://github.com/sponsors/nutti

**Comment by mac4kent** [comment]
Sure no problem.
